pages/bookmarks.jsx

```diff
@@ -86,12 +95,13 @@ export default React.createClass({
         });
       }
     });
-  },
+  }
+
   getContentForLoggedInUser() {
     let importHint;
 
     if (this.state.lsBookmarkedIds.length > 0) {
-      importHint = <span>Did you add favs while logged out? You can <button className="btn btn-link inline-link" onClick={evt=>this.handleImportBookmarksClick(evt)}>import favs</button> from this browser.</span>;
+      importHint = <span>Did you add favs while logged out? You can <button className="btn btn-link inline-link" onClick={(event) => this.handleImportBookmarksClick(event)}>import favs</button> from this browser.</span>;
```

Comment:
this did not need a refactor, so I'd strongly advise against changing this line as it isn't actually part of the work that this PR "is for": it does not change the behaviour of the code to be ES6 conformant, as it already was =)

If we want to change `evt` to `event` then we'll need to change that in a *lot* of other places, so let's roll back this particular change.